src/lib/components/01-elements/link/link.tsx

```diff
@@ -1,13 +1,76 @@
+import NextLink from 'next/link';
```

Comment:
Please take note of the required changes in the button component and also apply them here.